server/src/main/kotlin/com/kamelia/hedera/util/Extensions.kt

```diff
@@ -168,3 +187,30 @@ private const val CHARSET = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxy
 fun String.Companion.random(size: Int) = (1..size)
     .map { CHARSET.random() }
     .joinToString("")
+
+fun FilterObject.adaptFileSize(): FilterObject {
+    val (value, unit) = value.split(";")
+    val size = value.toDoubleOrNull() ?: throw IllegalFilterException(this)
+    val shift = unit.toIntOrNull() ?: throw IllegalFilterException(this)
+    val bytes = size * (1 shl shift)
+    return copy(value = bytes.roundToLong().toString())
+}
+
+/** Checks if this expression fuzzy matches the specified [pattern]. */
+infix fun <T : String?> Expression<T>.fuzzy(pattern: String) = fuzzy(LikePattern(pattern))
+
+/** Checks if this expression fuzzy matches the specified [pattern]. */
+infix fun <T : String?> Expression<T>.fuzzy(pattern: LikePattern): Op<Boolean> = when (currentDialect) {
+    is PostgreSQLDialect -> FuzzyMatchOp(this, stringParam(pattern.pattern))
+    else -> LikeEscapeOp(this, stringParam("%${pattern.pattern}%"), true, pattern.escapeChar)
+}
+
+class FuzzyMatchOp(private val expr1: Expression<*>, private val expr2: Expression<*>) :
+    Op<Boolean>(), ComplexExpression {
```

Comment:
```suggestion
class FuzzyMatchOp(
    private val expr1: Expression<*>, 
    private val expr2: Expression<*>,
) : Op<Boolean>(), ComplexExpression {
```

no cantor format here 😠

Reply:
¿ Hablas español ?